src/mlpack/methods/preprocess/load_save_image_main.cpp

```diff
@@ -0,0 +1,135 @@
+/**
+ * @file load_save_image_main.cpp
+ * @author Jeffin Sam
+ *
+ * A CLI executable to load and image dataset.
```

Comment:
I guess you meant "load and save an image dataset"?

Reply:
Thanks for pointing out